lib/katello/repository_types/python.rb

```diff
@@ -0,0 +1,17 @@
+require 'pulp_python_client'
+Katello::RepositoryTypeManager.register('python') do
+  allow_creation_by_user true
+  pulp3_service_class Katello::Pulp3::Repository::Generic
+  pulp3_api_class Katello::Pulp3::Api::Generic
+  pulp3_plugin 'python'
+  repositories_class PulpPythonClient::RepositoriesPythonApi
+  api_class PulpPythonClient::ApiClient
+  configuration_class PulpPythonClient::Configuration
+
+  content_type Katello::FileUnit,
```

Comment:
`content_type` and `default_managed_content_type` will need to be made generic as well. For now you can leave it all empty instead of having the file units be there.  It shouldn't cause issues with repo CRUD.

Reply:
For these guys, an object is a required argument. Is there a way of leaving them empty besides nil or excluding them? I left the File implementation there because it doesn't cause issues and I didn't think a generic version fits within the context of the card.

Reply:
@rverdile you could try removing `content_type` and `default_managed_content_type`.